spec/airport.spec.js

```diff
@@ -0,0 +1,603 @@
+import Airport from '../src/airport.js';
+import { assertEquals, assertFalse, assertThrows, assertTrue } from '../test_framework.js';
+
+console.log('------------------------------');
+console.log('Running tests for airport.js');
+let passCount = 0, failCount = 0;
+
+// Test 1
+{
+	console.log('------------------------------');
+	console.log('Test 1. (land() test). Empty airport, valid plane. => 1')
+
+	let input, expected, actual, result, airport, plane, weather, srandom;
+	
+	//1. Arrange
+	plane = {};
+	airport = new Airport({ getWeather: () => {}});
+	expected = 1;
+
+	//2. Act
+	airport.land(plane, true);
+	actual = airport.planes.length;
+
+	//3. Assert
+	result = assertEquals(expected, actual);
+	
+	if (result) {console.log("Test Passed."); passCount += 1;}
+	else {console.log(`Test Failed. Expected ${expected} but received ${actual}.`); failCount += 1;}
+	console.log('Output: ' + result);
+}
+
```

Comment:
Using Jasmine would also allow for testing if the correct method is being called, the correct number of times by spying on it.